docs/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,23 @@
+# 1. Record architecture decisions in manage-courses-backend
+
+Date: 2019-12-18
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made for the manage-courses and
```

Comment:
I'm really keen to get `manage` changed to `publish` but not a blocker for this

Reply:
I like that idea. The idea of merging all the repos into one came up again yesterday, esp in context of this. We may not be doing that yet, but probably worth keeping in mind if we’re looking to rename.